dev-libs/xxhash/xxhash-0.6.4.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A fast non-cryptographic hash algorithm"
+DESCRIPTION="xxHash is an extremely fast non-cryptographic hash algorithm, \
+working at speeds close to RAM limits."
+HOMEPAGE="https://cyan4973.github.io/xxHash/
+	http://www.xxhash.com/"
+COPYRIGHT="2012-2016 Yann Collet"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://github.com/Cyan4973/xxHash/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="4570ccd111df6b6386502791397906bf69b7371eb209af7d41debc2f074cdb22"
+SOURCE_FILENAME="xxHash-$portVersion.tar.gz"
+SOURCE_DIR="xxHash-$portVersion"
+PATCHES="xxHash-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Comment:
```diff
-ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="x86"
-SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+# Provide cmd:xxhash on all primary architectures except x86_gcc2,
+# as well as on x86 secondary arch if the primary arch is x86_gcc2.
+commandBinDir="$binDir"
+provideRuntime=yes
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir="$prefix"/bin
+else
+	if [ -n "$secondaryArchSuffix" ]; then
+		provideRuntime=no
+	fi
+fi
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+	provideRuntime=no
+fi
```

Remarks:
* `$secondaryArchSuffix` is empty when we build a recipe for a primary arch.
* `$effectiveTargetArchitecture` is `x86_gcc2` if you are on that arch and run `haikuporter xxhash`.
* `$effectiveTargetArchitecture` is `x86` if you are on that arch and run `haikuporter xxhash` **or** if you are on `x86_gcc2` and run `haikuporter xxhash_x86`.

Reply:
Fixed.